conda/recipes/libcugraph/recipe.yaml

```diff
@@ -0,0 +1,268 @@
+          - lib/libcugraph.so
+          - lib/libcugraph_c.so
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - rapids-logger =0.1
+        - if: cuda_major == "11"
+          then:
+            - cudatoolkit
+            - libcublas-dev ${{ cuda11_libcublas_host_version }}
+            - libcurand-dev ${{ cuda11_libcurand_host_version }}
+            - libcusolver-dev ${{ cuda11_libcusolver_host_version }}
+            - libcusparse-dev ${{ cuda11_libcusparse_host_version }}
+          else:
+            - cuda-cudart-dev
+            - libcublas-dev
+            - libcurand-dev
+            - libcusolver-dev
+            - libcusparse-dev
+      run:
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - libraft =${{ minor_version }}
+        - librmm =${{ minor_version }}
+        - nccl ${{ nccl_version }}
+        - if: cuda_major == "11"
+          then:
+            - cudatoolkit
+            - cuda-profiler-api ${{ cuda11_cuda_profiler_api_run_version }}
+            - libcublas ${{ cuda11_libcublas_run_version }}
+            - libcurand ${{ cuda11_libcurand_run_version }}
+            - libcusolver ${{ cuda11_libcusolver_run_version }}
+            - libcusparse ${{ cuda11_libcusparse_run_version }}
+          else:
+            - cuda-profiler-api
+            - cuda-cudart
+            - libcublas
+            - libcurand
+            - libcusolver
+            - libcusparse
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-nvtx
+          - cuda-version
+          - libcublas
+          - libcudf
+          - libcurand
+          - libcusolver
+          - libcusparse
+          - librmm
+          - libucxx
+          - nccl
+          - openmpi
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    about:
+      homepage: ${{ load_from_file("python/libcugraph/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libcugraph/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libcugraph/pyproject.toml").project.description }}
+
+  - package:
+      name: libcugraph_etl
+      version: ${{ version }}
+    build:
+      script:
+        content: |
+          cmake --install cpp/libcugraph_etl/build
+      dynamic_linking:
+        overlinking_behavior: "error"
+      prefix_detection:
+        ignore:
+          # See https://github.com/rapidsai/build-planning/issues/160
+          - lib/libcugraph_etl.so
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - libcudf =${{ minor_version }}
```

Comment:
Does this need `libcudf` in `run`? Check the linking, I think `libcugraph_etl.so` links to libcudf but I could be wrong.